src/couch_server.erl

```diff
@@ -91,11 +91,8 @@ open(DbName, Options0) ->
         end
     end.
 
-update_lru(DbName, Options) ->
-    case lists:member(sys_db, Options) of
-        false -> gen_server:cast(couch_server, {update_lru, DbName});
-        true -> ok
-    end.
+update_lru(DbName, _Options) ->
```

Comment:
Though it'll only be problematic if you force an upgrade mid-call to `couch_db:open/2`, right? Might be worth leaving in just so those don't crash, but it'd be nice to have a path going forward to remove that cruft.